src/UI/Movies/FullMovieCollection.js

```diff
@@ -1,6 +1,7 @@
 var movieCollection = require('./MoviesCollection');
 
 var fullCollection = movieCollection.clone();
+fullCollection.reset();
```

Comment:
@fedoranimus No because then Movie Controller thinks you have more than 0 models (since it has the 25 from the first page still) which causes 404s.